script/build-fuzzers

```diff
@@ -39,34 +39,39 @@ for lang in ${languages[@]}; do
 
   lang_scanner="${lang_dir}/src/scanner"
   if [ -e "${lang_scanner}.cc" ]; then
-    $CXX $CXXFLAGS -g -O1 "-I${lang_dir}/src" -c "${lang_scanner}.cc" -o "${lang_scanner}.o"
+    $CXX "$CXXFLAGS" -g -O1 "-I${lang_dir}/src" -c "${lang_scanner}.cc" -o "${lang_scanner}.o"
     objects+=("${lang_scanner}.o")
   elif [ -e "${lang_scanner}.c" ]; then
-    $CC $CFLAGS -std=c99 -g -O1 "-I${lang_dir}/src" -c "${lang_scanner}.c" -o "${lang_scanner}.o"
+    $CC "$CFLAGS" -std=c99 -g -O1 "-I${lang_dir}/src" -c "${lang_scanner}.c" -o "${lang_scanner}.o"
     objects+=("${lang_scanner}.o")
   fi
 
 
   # Compiling with -O0 speeds up the build dramatically
-  $CC $CFLAGS -g -O0 "-I${lang_dir}/src" "${lang_dir}/src/parser.c" -c -o "${lang_dir}/src/parser.o"
+  $CC "$CFLAGS" -g -O0 "-I${lang_dir}/src" "${lang_dir}/src/parser.c" -c -o "${lang_dir}/src/parser.o"
```

Comment:
Wrapping vars in double quotes breaks variables expansion in the shell in case if there are multiple parameters delimited with spaces. Please remove all added double quote wrappings.

Reply:
sure